config.yaml

```diff
@@ -402,3 +402,7 @@ options:
       Allowed values are: from 0 to 2000000000.
     type: int
     default: 150000000
+  experimental_max_connections:
+    type: int
+    description: |
+      [EXPERIMENTAL] Force set max_connections.
```

Comment:
Should we add some kind of validation here?

Reply:
should we maybe enforce an upper limit?

Reply:
It's a footgun config.